src/scripts.js

```diff
@@ -51,12 +51,38 @@ const hydrationDate = document.querySelector('#hydrationDate');
 const hydrationResponse = document.querySelector('#hydrationResponse');
 const hydrationForm = document.querySelector('#hydrationForm');
 
+const stepSwivelButton = document.querySelector('#stepSwivelButton');
```

Comment:
Also instead of using so many querySelectors we can do something like:

const swivelButton = document.querySelectorAll('.swivel-button');

and that would be used instead in the forEach i provided below.